bundles/redhat-marketplace/4.0.4/manifests/minio-directpv-operator-rhmp.clusterserviceversion.yaml

```diff
@@ -272,7 +272,7 @@ spec:
                       fieldPath: spec.nodeName
                 - name: CSI_ENDPOINT
                   value: unix:///csi/csi.sock
-                image: quay.io/minio/directpv@sha256:1f978ff8d13ba6ce9f11110218ab227545813c3d3003a7b489d483706637e7ea
+                image: registry.connect.redhat.com/minio/directpv@sha256:53610c35d42971ad57ce8491dbcab3b95395a68820d9024a1298b9f653802688
```

Comment:
@cniackz We have two certified images at https://catalog.redhat.com/software/containers/minio/directpv/64a595e8754135d767c5c106?tag=all. We would need to generate bundles for `4.0.6` (latest), `4.0.5` if needed, and remove `4.0.3` and `4.0.4` bundles.

The new bundles can be generated by
```sh
$ ./olm.sh 4.0.6 sha256:53610c35d42971ad57ce8491dbcab3b95395a68820d9024a1298b9f653802688

$ ./olm.sh 4.0.5 sha256:e3252e8004a966cbc248159b359f3ce40533aa1e83cc126fb1cf2ed6b130efb6
```

If possible, we can skip `4.0.5` bundles as we always recommend to use the latest version.